src/coffee/stockimport.coffee

```diff
@@ -90,7 +89,7 @@ class StockImport
         ElasticIo.returnFailure err, next
       .done()
     else
-      ElasticIo.returnFailure "#{LOG_PREFIX}No data found in elastic.io msg.", next
+      ElasticIo.returnFailure "#{CONS.LOG_PREFIX}No data found in elastic.io msg.", next
```

Comment:
Whoops, I hope not, it took me some time to get everything working with `elastic.io` after the changes...